app/resources/create.html.template

```diff
@@ -40,15 +40,10 @@
               {% endifequal %}
             </h1>
           </div>
-          <div class="name section">
+          <div class="identifyinfo section">
```

Comment:
Maybe just "identity" is OK. Or it should be "identifying-info".

You should avoid abbreviations unless it's commonly used e.g., info (for readability). And you should put a dash "-" between words for class names. (HTML class/id names in Person Finder is very inconsistent, so you would see other namings too :P)